Cargo.toml

```diff
@@ -62,6 +62,8 @@ members = [
   "zkp-circuits",
 ]
 
+resolver = "2"
```

Comment:
Not sure if we want to set this or set the `edition` :thinking:

Reply:
I think Sören is right, so setting the resolver is good for now